### ISSUE TITLE
Select adds 'second' scrollbar

### ISSUE THREAD

**Issue by ju5t** [livewire/flux#844]
### Flux version

v1.0.29

### Livewire version

v3.5.12

### What is the problem?

This is a weird one. The issue appeared after upgrading to v1.0.29.

Our `body`-tag has `overflow-y-scroll`. I don't know why.

```html
<body class="overflow-y-scroll bg-red-900">
    <flux:select variant="listbox" placeholder="Choose industry...">
        <flux:option>Photography</flux:option>
        <flux:option>Design services</flux:option>
        <flux:option>Other</flux:option>
    </flux:select>
    @fluxScripts
</body>
```

This renders a scrollbar, you can see a gap on the right. In our app itself the scrollbar does show, but in my screenshot it doesn't for some reason.

![Image](https://github.com/user-attachments/assets/d13cac66-8426-4598-b877-3f9ccda2740b)

Whenever I open the select, it adds a second scrollbar:

![Image](https://github.com/user-attachments/assets/4250cb4f-4268-4764-bc79-ea27db563c14)

Removing `overflow-y-scroll` solves the issue for us so feel free to close this. If anything it helps others coming across the same thing in the future.

### Please confirm (incomplete submissions will not be addressed)

- [x] I have provided easy and step-by-step instructions to reproduce the bug.
- [x] I have provided code samples as text and NOT images.
- [x] I understand my bug report will be closed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@ju5t I've had a look into this, and what is happening is when the scroll lock is applied, when a dropdown is open, these styles `style="padding-right: 7px; overflow: hidden;"` get applied to the `<html>` element.

I'm not exactly sure why there is a padding right being added.

What should also be happening is it would normally detect what overflow type you have, store it, remove it and apply overflow hidden. But your overflow is being applied to the body tag, and the scroll lock is being applied to the body tag, hence why the "double scroll" gap is happening (one is your body scroll bar and the other is the padding right).

Maybe @calebporzio can shed some light on why the padding right and why it's being applied to the html tag instead of body, as I would typically apply stuff to the body tag as you have.

**Comment by calebporzio** [comment]
Yeah good explanation Josh - the padding is there because applying hidden typically removes the scrollbar gutter and causes a page layout shift and adding padding fixes that.

In the future we want to use the new scrollbar preserve css stuff and this won't be an issue. For now, please just remove overflow-y-scroll. thanks!
